### PR TITLE
Updated the Contributor Guide

### PR DESCRIPTION
**Pull request by n-boshnakov**
**What this PR does / why we need it**:
This PR updates the Contributor Guide:
- adds new content - Images page
- removes outdated content - Community page
- updates existing content - Markdown and Style Guide pages, Topic templates, Community page
- unifies the style of the Code and Documentation sections

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement developer
added new guidelines for image creation when contributing content to the documentation
```

### CODE REVIEW

File: website/community/_index.md [hunk 1]
```diff
@@ -25,3 +25,6 @@ The recordings are published on the [Gardener Project YouTube channel](https://w
 <p style="margin-top:30px"><strong>Google Calendar</strong></p>
 <iframe src="https://calendar.google.com/calendar/embed?src=gardener.cloud.community%40gmail.com&ctz=Europe%2FBerlin" style="border: 0;margin-bottom:30px" width="800" height="600" frameborder="0" scrolling="no"></iframe>
 
+### Presenting a Topic
+
+If there is a topic you would like to present, message us in our slack channel or get in touch with [Jessica Katz](mailto:jessica.katz@sap.com).
```
Comment (JordanJordanov) [inline website/community/_index.md:30]: message us in our slack channel -> shall we name the slack channel  (e.g. \#gardener) and make it a link: https://kubernetes.slack.com/archives/CB57N0BFG? Or this info is already in the page?

File: website/documentation/contribute/20_documentation/10_organisation/_index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The Gardener project implements the *documentation-as-code* paradigm. Essentiall
 
 Documentation that can be affiliated to component is hosted and maintained in the component repository.
 
-A recommended template for organizing documentation sources is to place them all in a `docs` folder and organize it there per role activity. For example:
+A template for organizing documentation sources is to place them all in a `docs` folder and organize it there per role activity. For example:
```
Comment (JordanJordanov) [inline website/documentation/contribute/20_documentation/10_organisation/_index.md:17]: What about: A good way to organize your documentation is to place it in a 'docs' folder and create separate subfolders per role activity, For eample:

File: website/community/_index.md (hunk 1 above)
Comment (n-boshnakov) [reply website/community/_index.md:30]: Screenshot 2022-10-19 144255
The link to the Gardener slack channel would be right below the content, which is why I decided not to include it. It won't be an issue if we add it, though.

**Review by JordanJordanov — APPROVED** [review]
/lgtm

**Pull request merged**